apps/golden-sample-app/src/app/user-context/user-context.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { LOGOUT } from '@backbase/foundation-ang/web-sdk';
+import { OAuthService } from 'angular-oauth2-oidc';
+
+const logoutFactoryService = (oAuthService: OAuthService) => {
+  return {
+    goToLoginPage: () => oAuthService.initLoginFlow(),
+    logout: () => oAuthService.revokeTokenAndLogout(),
+  };
+};
+
+@Component({
+  selector: 'app-user-context',
+  templateUrl: './user-context.component.html',
+  providers: [
+    {
+      provide: LOGOUT,
+      useFactory: logoutFactoryService,
+      deps: [OAuthService],
```

Comment:
Can we move deprecated providers to app.module better?

Reply:
I thought that it is better to keep it as close to the code that actually requires it.